images/wkdev_sdk/Containerfile

```diff
@@ -145,5 +141,11 @@ RUN gst-inspect-1.0 audiornnoise && \
     gst-inspect-1.0 livesync && \
     gst-inspect-1.0 rsrtp
 
+# Remove systemd services that would startup by default, when spawning
+# systemd as PID 1 within the container (usually, we don't spawn systemd
+# within the wkdev-sdk container, for interactive usage, but we do so
+# when deploying the wkdev-sdk image as bot).
+RUN find /etc/systemd/ -type l -name '*.service' -print -exec rm {} \;
```

Comment:
I do worry about this causing an easy to miss issue in the future, however looking at the current services there I don't think there is a problem.

Reply:
Also it shouldn't be a problem to actually remove things like cupsd (socket activated anyway) and pipewire from the image, we only use them as clients, if the packages allow that.

Use do use httpd for some testing though.

Reply:
OK, we can always refine. This is most likely not the last change of this kind.